app/locales/se.json

```diff
@@ -0,0 +1,209 @@
+  "required": "nödvändigt",
+  "recommended": "rekommenderat",
+  "Below is your personal health information.": "Nedan visas din hälsoinformation.",
+  "If you want to register another person on this computer, save or write down your personal link":
+  "För att lägga till en annan person på denna dator, spara eller skriv ner din personliga länk",
+  "Your personal health information is cleared from this computer.":
+  "Din hälsoinformation är rensad från datorn.",
+  "You can now use this computer for another user.": "Du kan nu använda den här datorn för en annan användare.",
+  "We stored your data in a cookie so come back later to update your status":
+  "Vi har sparat din data i en cookie tills nästa gång du vill uppdatera din status",
+  "If you use this computer with multiple users, please use this link to update your status.":
+  "Om du har flera användare på den här datorn, var god använd den här länken för din status.",
+  "Write it down somewhere or save it on your computer.": "Skriv ner den eller spara den på din dator.",
+  "The link has been copied.": "Länken har kopierats.",
+  "clear this form": "rensa formuläret",
+  "https://who.int": "https://who.int",
+  "To WHO.int": "Till WHO.int",
+  "Are you from": "Bor du i",
+  "Sweden": "Sverige",
+  "Norway": "Norge",
+  "The Netherlands": "Nederländerna",
+  "Slovakia": "Slovakien",
+  "United Kingdom": "Storbritannien",
+  "March 16": "16 mars",
+  "March 17": "17 mars",
+  "March 23": "23 mars",
+  "at": "klockan",
+  "Nausea or vomiting": "Illamående eller kräkningar",
+  "Have you been diagnosed with any of the following conditions?":
+  "Har du blivit diagnostiserad med något av följande sjukdomstillstånd?",
+  "Diabetes mellitus, Hypertension, Ischemic heart disease, Asthma, Chronic lung disease, Chronic kidney disease":
+  "Diabetes, högt blodtryck, hjärt- och kärlsjukdom, astma, kronisk lungsjukdom, kronisk njursjukdom",
+  "I am currently:": "Jag befinner mig för närvarande",
+  "Not in isolation": "Inte i karantän",
+  "In isolation due to a recent international travel": "I karantän pga nylig utlandsresa",
+  "In isolation due to contact with an individual who has been exposed to the coronavirus.":
+  "I karantän på grund av en kontakt som blivit utsatt för coronaviruset.",
+  "Cigarette smoking habits:": "Rökvanor",
+  "I currently smoke": "Jag är rökare",
+  "I used to smoke": "Jag har tidigare varit rökare",
+  "I have never smoked": "Jag har aldrig varit rökare",
+  "What is your current body temperature? (Celcius)": "Vad är din kroppstemperatur, i grader Celsius?",
+  "This field is optional": "Detta fält är frivilligt",
+  "Close": "Stäng",
+  "Contribute now": "Bidra nu",
+  "Help us uncover and predict the corona spread in your area":
+  "Hjälp oss <span>kartlägga</span> och <span>förutse</span> spridningen av corona i ditt område"
```

Comment:
There are a couple of new translations on master that you are missing, but you can just add them in a new PR.

Reply:
Thanks for point this out, done in #421